test/api/v3/unit/models/group.test.js

```diff
@@ -556,47 +546,47 @@ describe('Group Model', () => {
           emails.push(`user-${i}@example.com`);
         }
 
-        expect(function () {
-          Group.validateInvitations(uuids, emails, res);
+        expect(async function () {
```

Comment:
these `expect()...to.not.throw()` aren't working correctly with `async` functions. If you change the code to

```
      it.only('does not throw an error if emails are passed in and uuids are an empty array', async () => {
        expect(async function () {
          throw new Error('error');
          await Group.validateInvitations([], ['user1@example.com', 'user2@example.com'], res);
        }).to.not.throw();

        expect(res.t).to.not.be.called;
      });
```

the test passes even if it should not.

But you don't need to expect the function not to throw because if an error is thrown inside an async function the function and the test fails. So this will work

```
      it.only('does not throw an error if emails are passed in and uuids are an empty array', async () => {
        await Group.validateInvitations([], ['user1@example.com', 'user2@example.com'], res);
        expect(res.t).to.not.be.called;
      });
```